adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -119,6 +119,16 @@ public Agency getAgencyById(final String agencyId) {
         return agencyPerId.get(agencyId);
     }
 
+    /**
+     * Return a collection of Agency objects representing all the rows from agency.txt
+     *
+     * @return a collection of Agency objects representing all the rows from agency.txt
+     */
+    @Override
+    public Collection<Agency> getAgencyAll() {
+        return agencyPerId.values();
```

Comment:
Shall we apply the principle of exposing only an immutable version? I am not sure that's the case here

Reply:
Absolutely. Indeed, here this is not the case. Modifying this.